sunpy/map/sources/hinode.py

```diff
@@ -36,24 +47,31 @@ class XRTMap(GenericMap):
                                   "Be_thick", "Gband", "Ti_poly"]
 
     def __init__(self, data, header, **kwargs):
-        super().__init__(data, header, **kwargs)
-
-        fw1 = header.get('EC_FW1_')
+        fw1 = header.get('EC_FW1_', '')
         if fw1.lower() not in _lower_list(self.filter_wheel1_measurements):
-            raise ValueError('Unpexpected filter wheel 1 in header.')
-
-        fw2 = header.get('EC_FW2_')
+            raise ValueError(f'Unexpected filter wheel 1 {fw1} in header.')
+        fw2 = header.get('EC_FW2_', '')
         if fw2.lower() not in _lower_list(self.filter_wheel2_measurements):
-            raise ValueError('Unpexpected filter wheel 2 in header.')
-
+            raise ValueError(f'Unexpected filter wheel 2 {fw2} in header.')
+        super().__init__(data, header, **kwargs)
         self.plot_settings['cmap'] = 'hinodexrt'
 
     @property
     def _timesys(self):
         if self.meta.get('timesys', '').upper() == 'UTC (TBR)':
             return 'UTC'
-        else:
-            return super()._timesys
+        return super()._timesys
+
+    @property
+    def _supported_observer_coordinates(self):
+        # Assume observer is at zero Stonyhurst heliographic longitude if not otherwise specified
+        # https://community.openastronomy.org/t/sunpymetadatawarnings-when-using-hinode-xrt-data/393/7 for more information
+        return (super()._supported_observer_coordinates
+                + [(('solar_b0', 'dsun_obs'), {'lon': 0*u.deg,
+                                               'lat': self.meta.get('solar_b0'),
+                                               'radius': self.meta.get('dsun_obs'),
+                                               'unit': (u.deg, u.deg, u.m),
+                                               'frame': "heliographic_stonyhurst"})])
```

Comment:
Is `'solar_b0'` in the meadata, so we don't actually need to calculate it ourselves?

Reply:
In the level 1 data it should be, we don't have a level 1 header at the moment.